### DISCUSSION TITLE
Language localization is not applied (missing whitelist)

### DISCUSSION THREAD

**Discussion by kzsa** [netblue30/firejail#5974] [Q&A] [answered]
Hello everyone,
does anyone know why the following change affects I18N & L10N? When I use such a modification, several programs switch to "default English". 
e.g:
```
~$ cat .config/firejail/gnome-calculator.local 
private-bin gnome-calculator,sh
ignore include disable-shell.inc
```
Thx: Zsolt
( For many programs, using the default setting will display the applications in the appropriate language.)

**Comment by ghost** [comment]
No clue as to how/why your gnome-calculator.local affects I18N & L10N. My only advice right now is to double-check what's in your `~/.config/user-dirs.locale`. You can create that file with your preferred locale if it's missing.

https://wiki.archlinux.org/title/XDG_user_directories

**Comment by kzsa** [comment, marked as answer]
> No clue as to how/why your gnome-calculator.local affects I18N & L10N. My only advice right now is to double-check what's in your `~/.config/user-dirs.locale`. You can create that file with your preferred locale if it's missing.
> 
> https://wiki.archlinux.org/title/XDG_user_directories

Thank you, I found it!

Here is the source of the problem:
$ cat /etc/firejail/whitelist-usr-share-common.inc
The file does not contain the "whitelist /usr/share/locale-langpack/" permission.

commit e07df677c5ee2e3b46d77874533bfda3bb45d89e

**Reply by ghost** [reply]
Nice! Thanks for the related PR.

**Reply by kzsa** [reply]
My pleasure! It is an honour for me to be involved, even if it is just a small thing. :)

**Reply by ghost** [reply]
Firejail is very much a community-based project. Complexities are many and your contribution - however large or small - does improve future sandboxing. Always a pleasure receiving feedback/PR's from active users :).

**Comment by kzsa** [comment]
To tell the full story:
https://github.com/linuxmint/cinnamon/issues/11777#issue-1817276432
I am therefore amending the title and I think it can be closed.

**Comment by kmk3** [comment]
Should be fixed by:

* #5981
